hack/dockerfile/install-binaries.sh

```diff
@@ -68,11 +85,11 @@ do
 			;;
 
 		proxy)
-			echo "Install docker-proxy version $LIBNETWORK_COMMIT"
-			git clone https://github.com/docker/libnetwork.git "$GOPATH/src/github.com/docker/libnetwork"
-			cd "$GOPATH/src/github.com/docker/libnetwork"
-			git checkout -q "$LIBNETWORK_COMMIT"
-			CGO_ENABLED=0 go build -v -o /usr/local/bin/docker-proxy github.com/docker/libnetwork/cmd/proxy
+			CGO_ENABLED=0 install_proxy
+			;;
+
+		proxy-dynamic)
+			CGO_ENABLED=1 PROXY_LDFLAGS="-linkmode external" install_proxy
```

Comment:
An equal sign is missing, it should be `-linkmode=external`. Might not be super important but it's safer, also it's how it was previously.

And do you confirm that it doesn't work with cgo disabled `CGO_ENABLED=0 PROXY_LDFLAGS='-linkmode=external'` ?